src/RevMem.cc

```diff
@@ -755,31 +754,24 @@ bool RevMem::ReadMem( unsigned Hart, uint64_t Addr, size_t Len, void* Target, co
   char*    BaseMem     = &physMem[physAddr];
   char*    DataMem     = static_cast<char*>( Target );
 
+  // TODO: simplify to not consider crossing pages when using MemH
```

Comment:
Is there a reason you don't want to do that in this PR?

Reply:
> Is there a reason you don't want to do that in this PR?

This is a bugfix PR and I don't want to risk a non-bugfix change; I also wanted to consult with you or others before changing how the code handles crossing page boundaries.